### ISSUE TITLE
Update `CircularOptionPicker` consumers to use `asButtons`

### ISSUE THREAD

**Issue by andrewhayward** [WordPress/gutenberg#55013]
## What problem does this address?

In #35292 the behaviour of `CircularOptionPicker` was changed from a set of buttons to a `listbox`. It was later noticed (#54239) that sometimes this was not desired, and work was done (#54290) to allow `CircularOptionPicker` to act as a set of buttons where necessary.

## What is your proposed solution?

All relevant `CircularOptionPicker` consumers should be identified and updated to pass `asButtons`, to ensure that the UI behaves as expected in context.

## Acceptance Criteria

 - [x] Identify all of the controls that use `CircularOptionPicker` [^1]
 - [ ] Identify which of these should behave as buttons rather than a list box
 - [ ] Update these to use `asButtons`

[^1]: This will actually mean consumers of `ColorPalette`, `DuotonePicker` and `GradientPicker`, which has been done in #54055.

**Comment by youknowriad** [comment]
I'm closing this issue as stale. The new component package is likely to start from scratch here.
